tests/e2e/network/network_security_group.go

```diff
@@ -162,6 +162,29 @@ var _ = FDescribe("Network security group", func() {
 		ipList := []string{ip1, ip2}
 		Expect(validateSharedSecurityRuleExists(nsg, ipList, port)).To(BeTrue(), "Security rule for service %s not exists", serviceName)
 	})
+
+	It("can set source IP prefixes automatically accroding to corresponding service tag", func() {
+		By("Creating service and wait it to expose")
+		annotation := map[string]string{
+			azure.ServiceAnnotationAllowedServiceTag: "Internet",
```

Comment:
`Internet` is the default service tag, could we change it to a different one, e.g. `AzureCloud`?

Reply:
Sure. Thanks for reminding.